### ISSUE TITLE
Suggested enhancements to LeadingZero and EmptyLineBetweenBlocks?

### ISSUE THREAD

**Issue by chris-dura** [brigade/scss-lint#92]
If I could write Ruby, I'd probably just submit a PR as I imagine these "enhancements" are fairly easy to implement. But, I'm just a lowly designer and I haven't even done a "Hello, World!" in Ruby, yet, so thought I just raise an issue to see take the temperature of the room?
### LeadingZero

The internal standards of my current team actually require the use of leading zeros, e.g. `opacity: 0.1;` instead of `opacity: .1;`
I understand not having them is simpler both programmatically, and you save a bit (or is it byte?). But, `.scss` files are written by humans, and my subjective opinion is I _feel_ that `0.1` is easier to read for humans. A quick poll of the dev team upstairs gave me a pretty close to 50/50 split for/against. I imagine it would be easy to solve with a config parameter like you do with the Indentation linter, either you want leading zeros or you don't.
### EmptyLineBetweenBlocks

My team has allowed for a specific formatting exception to this rule where we allow for single-line rules without empty lines between when it makes sense or aides in legibility or just makes editing inline a bit easier.

For example:

``` scss
.icon-chevron-up          { &:before { content: "\e030"; } }
.icon-chevron-down        { &:before { content: "\e031"; } }
.icon-chevron-left        { &:before { content: "\e032"; } }
.icon-chevron-right       { &:before { content: "\e033"; } }
```

It'd be nice (maybe just for me) if there was an easy way to make single-line blocks exempt from the EmptyLineBetweenBlocks linter.
